examples/resources/configuration/certificates.yaml

```diff
@@ -0,0 +1,588 @@
+              "type": "string",
+              "name": "duration"
+            },
+            {
+              "path": "spec.encodeUsagesInRequest",
+              "type": "boolean",
+              "name": "encodeUsagesInRequest"
+            },
+            {
+              "path": "spec.isCA",
+              "type": "boolean",
+              "name": "isCA"
+            },
+            {
+              "path": "spec.renewBefore",
+              "type": "string",
+              "name": "renewBefore"
+            },
+            {
+              "path": "spec.revisionHistoryLimit",
+              "type": "integer",
+              "name": "revisionHistoryLimit"
+            },
+            {
+              "path": "spec.secretName",
+              "type": "string",
+              "name": "secretName"
+            },
+            {
+              "path": "status.conditions[?(@.type==\"Ready\")].status",
+              "type": "string",
+              "name": "Ready"
+            },
+            {
+              "path": "spec.secretName",
+              "type": "string",
+              "name": "Secret"
+            },
+            {
+              "path": "spec.issuerRef.name",
+              "type": "string",
+              "name": "Issuer"
+            },
+            {
+              "path": "status.conditions[?(@.type==\"Ready\")].message",
+              "type": "string",
+              "name": "Status",
+              "widget": "Badge"
+            },
+            {
+              "path": "metadata.creationTimestamp",
+              "type": "date",
+              "name": "Age"
+            }
+          ]
+        },
+        {
+          "path": "spec",
+          "widget": "CodeViewer"
+        }
+      ]
+    }
+  form: |
+    - path: spec.commonName
+    - path: spec.issuerRef
+      widget: FormGroup
+    - path: spec.issuerRef.group
+    - path: spec.issuerRef.kind
+    - path: spec.issuerRef.name
+    - path: spec.duration
+    - path: spec.encodeUsagesInRequest
+    - path: spec.isCA
+    - path: spec.renewBefore
+    - path: spec.revisionHistoryLimit
+    - path: spec.secretName
+    - path: spec.issuerRef
+      widget: FormGroup
+    - path: spec.issuerRef.kind
+    - path: spec.issuerRef.name
+  list: |-
+    [
+      {
+        "path": "spec.commonName",
+        "type": "string"
+      },
+      {
+        "path": "spec.issuerRef",
+        "widget": "ControlledBy",
+        "kind": "Issuer",
+      },
+      {
+        "path": "status.expirationDate",
+        "type": "string"
+      },
+      {
+        "highlights": {
+          "positive": ["Ready"],
+        },
+        "name": "status",
+        "path": "status.conditions[?(@.type==\"Ready\")].message",
+        "type": "string",
+        "widget": "Badge"
+      },
+    ]
+  resource: |-
+    {
+      "kind": "Certificate",
+      "group": "cert-manager.io",
+      "version": "v1",
+      "path": "example-certificates",
+      "scope": "namespace"
+    }
+  schema: |-
```

Comment:
As this [PR ](https://github.com/kyma-project/busola/pull/1634)is merged, we can delete the schemas from Config Maps.